### PR TITLE
Microsoft Teams Rich Presence

### PR DESCRIPTION
**Pull request by MatStef132**
Hello!
I have made rich presence for microsoft teams. It supports all (I think) pages of microsoft teams, except of course extensions and manager/teacher/admin only things.
Example screenshot of one of it: https://imgur.com/a/tDwvdEm
Screenshot of settings: https://imgur.com/a/rfuPRuE

### CODE REVIEW

File: websites/T/Teams/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  },
+  "url": "teams.microsoft.com",
+  "description": {
+    "en": "Stay connected and organized. Accomplish more together across work, school, and life with Microsoft Teams.",
+    "de": "Bleiben Sie gut vernetzt und organisiert – und erreichen Sie gemeinsam mehr mit Microsoft Teams: am Arbeitsplatz, in Schule und Studium und privat.",
+    "ru": "Будьте на связи и планируйте свои действия. Решайте поставленные задачи сообща и успевайте больше на работе, в школе и дома благодаря Microsoft Teams.",
+    "pl": "Bądź w kontakcie z innymi i zachowaj dobrą organizację pracy. Osiągaj więcej razem z innymi — w pracy, szkole i życiu — dzięki usłudze Microsoft Teams."
+  },
+  "service": "Microsoft Teams",
+  "version": "1.0",
```
Comment (Bas950) [inline websites/T/Teams/dist/metadata.json:15]: 1.0.0

**Review by Bas950 — CHANGES_REQUESTED** [review]
Also what you did with messages is meh, we are almost ready to beta release an update that adds the ability  to translate presences. So you might want to wait for that.

File: websites/T/Teams/dist/metadata.json (hunk 1 above)
Comment (MatStef132) [reply websites/T/Teams/dist/metadata.json]: Also to the messages I wanted an easy way for everyone to translate or change the text of the rich presence to how they want it.

**Comment by Timeraa** [comment]
> Also what you did with messages is meh, we are almost ready to beta release an update that adds the ability  to translate presences. So you might want to wait for that.

It's not meh, I'll add something like that to YouTube Music etc. too since you'll be able to change a lot of texts

**Review by Timeraa — APPROVED** [review]

**Comment by Bas950** [comment]
> It's not meh, I'll add something like that to YouTube Music etc. too since you'll be able to change a lot of texts. In terms of translations only, yes but in terms of customization nah

I mean for YouTube music you should do it that way for the format it uses when listening to a song. But I personally wouldn't add it for general text, I would use the translations.

**Review by Bas950 — APPROVED** [review]

**Pull request merged**